content/about/gsoc.rst

```diff
@@ -0,0 +1,27 @@
+Google Summer of Code
+=====================
+:slug: about/gsoc
+:title: Google Summer of Code
+:menu: Student Experience, Google Summer of Code, 5
+
+.. image:: /images/gsoc.jpg
+    :scale: 100%
+    :align: right
+    :alt: Google Summer of Code
+
+The Google Summer of Code program with the Oregon State University Open Source
+Lab is a yearly program designed to pair accepted student applicants with
+trained mentors in an effort to generate as many programming projects as our
+systems can handle. It is part of a larger initiative from Google to connect
+Open Source programs around the globe to create an international community
+through code. The OSU Open Source Lab uses this project as a way to introduce
+our own students to this international community while improving our existing
+systems through collaborative and original programming strategies.
```

Comment:
I think we need to be clear that this program is not something we created, only that we participate as a mentoring organization, and have been for many years.